### DISCUSSION TITLE
`Too many callbacks to handle in this frame warning

### DISCUSSION THREAD

**Discussion by meoowe** [korlibs/korge#2054] [Q&A] [answered]
Hello, whenever I run my game using runJvm I get the Too many callbacks to handle in this frame warning every 1 second. 
Here is the full error
```kotlin
korlibs.logger.DefaultLogOutput output
WARNING: Too many callbacks to handle in this frame elapsedTime=167ms, availableTime=167ms pending timedTasks=3, tasks=1, processedTimedTasks=3, processedTasks=1453690
2023-12-01T16:16:36.353Z[Thread[#63,AWT-EventQueue-0,6,main]]: WARNING: Korgw.GameWindow.TooManyCallbacks - Too many callbacks to handle in this frame elapsedTime=167ms, availableTime=167ms pending timedTasks=3, tasks=1, processedTimedTasks=2, processedTasks=1458060
``` 
Sorry to disturb everyone again

**Comment by maik-mursall** [comment, marked as answer]
Heya 👋🏻 
Don't worry too much, since this is only an warning and no error :)

This basically means there was too much going in this Frame, which could indicate a performance bottleneck within your App.

Keep in mind that the dev build will always perform much worse, than the release build 😉
